### PR TITLE
Retrieving comments as JSON objects and display it

### PR DESCRIPTION
**Pull request by riku1212**
Modified the following files:

1. pom.xml
- add GSON dependency

2. style.css
- add comment-container

3. index.html
- add div to display comments

4. script.js
- modify function to fetch comments from /data as JSON and make them into list element to display

5. DataServlet.java
- add hard-coded comments 
- add function to convert to JSON using GSON

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -12,21 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
```
Comment (kai-dream) [inline portfolio/src/main/webapp/script.js:15]: Thanks for deleting unused code. Keeping code clean is a good practice.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -4,30 +4,30 @@
...
-    text-align:justify;
+  margin-bottom: 20px;
+  text-align:justify;
 }
 
 .center {
   display: block;
-  margin-left: auto;
-  margin-right: auto;
+    margin-left: auto;
```
Comment (kai-dream) [inline portfolio/src/main/webapp/style.css]: Nitpick: Extra indentation.
Doesn't block this change, can make it in next commit.

**Review by kai-dream — APPROVED** [review]
Looks good. LTGM +1

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (riku1212) [reply portfolio/src/main/webapp/script.js:15]: thanks!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (riku1212) [reply portfolio/src/main/webapp/style.css]: corrected in new PR

**Pull request merged**